### ISSUE TITLE
US Data by States Now Unavailable

### ISSUE THREAD

**Issue by Allan-Avila** [CSSEGISandData/COVID-19#1549]
Will, you no longer be making the US data available by states?

This is crucial for local forecasts and determining which US regions are expected to increase the most.

**Comment by jaahumadap** [comment]
I agree! This is a critical piece of information. Can someone please update the community of users when will this be available again? I hope it's not the government trying to hide the data...

**Comment by mmacvicar-splunk** [comment]
See https://github.com/CSSEGISandData/COVID-19/issues/1250 for updates.

**Comment by becare-rocket** [comment]
I don't see these State timeseries data available.  It should be in files with the names:

time_series_covid19_confirmed_US.csv, 
time_series_covid19_deaths_US.csv, 
time_series_covid19_testing_US.csv, 

But I don't see these when browsing the JHU github.

Does anyone else see them?

**Comment by davidbau** [comment]
Duplicate of #1527

**Comment by davidbau** [comment]
Duplicate of #1505 too.

**Comment by rks125** [comment]
You can temporarily get what you are looking for here. I have posted this on other threads. 

https://www.soothsawyer.com/john-hopkins-time-series-data-confirmed-case-csv-after-march-22-2020/?guthub=2

**Comment by bfosten** [comment]
In case it's helpful for folks who are encountering this issue, we've set up a new repo, [CovidAPI](https://github.com/coviddata/covid-api), which provides time series data for states and provinces (see [Regions](https://github.com/coviddata/covid-api#regions)), as well as other formats of the Johns Hopkins data.
